src/blenderbim/blenderbim/bim/module/spreadsheet_writer/operator.py

```diff
@@ -0,0 +1,652 @@
+import os
+import sys
+import time
+import site
+import collections
+import subprocess
+
+site.addsitedir(os.path.join(os.path.dirname(os.path.realpath(__file__)), "libs", "site", "packages"))
+
+import bpy
+from bpy.props import StringProperty, BoolProperty, IntProperty, EnumProperty
+from bpy_extras.io_utils import ImportHelper 
+from bpy.types import (Operator, PropertyGroup)
+
+import blenderbim.bim.import_ifc
+from blenderbim.bim.ifc import IfcStore
+import blenderbim.tool as tool
+import ifcopenshell
+
+import openpyxl
+from openpyxl import load_workbook
+import pandas as pd
+import xlsxwriter
```

Comment:
These few lines will throw an error on startup if the user doesn't have these packages installed in their environment. Is the plan to distribute openpyxl and xlsxwriter with blenderbim ? (inside site-packages)

Reply:
I thought about completely removing the xlsx functionality, since this is not really an open-source and free solution.